tools/build_config/Jenkinsfile

```diff
@@ -192,13 +192,16 @@ pipeline {
 	    if (isUnix()) {
```

Comment:
We could pull the CI requirements automagically in from a requirements file like you've created. We could parse a very simple syntax which is human readable (kind of YAML) and just gets the appropriate things. Just an idea as really low priority and we have so few requirements.

```
pip:
    sphinx >= 4.0.0
    sphinx_rtd_theme
    sphinxcontrib-matlabdomain
git:
    ?
```

Reply:
Yeah, that would be a good idea. For the `git` part - do you mean stuff for the `module load` commands?

Also, how are we going to do things on Windows with the new ISIS cloud machine instead of Alex's box? (Chocolatey?)